config/routes.rb

```diff
@@ -1,3 +1,9 @@
 Rails.application.routes.draw do
-  resources :recycle_locations
+  resources :admins
+  resources :recycle_locations do
+    resources :recycle_location_changes, only: :create
+  end
+
+  get '/get_pending_changes', to: 'recycle_locations#get_pending_changes'
```

Comment:
`/pending_changes` already indicates that it will return a list of pending changes, which makes `get_` part  unnecessary while defining a routes rule :)